rfcs/0036-rfc-process-team-amendment.md

```diff
@@ -0,0 +1,202 @@
+---
+feature: rfc-process-team-amendment
+start-date: 2018-10-27
+author: Robin Gloster <mail@glob.in>
+co-authors: Graham Christensen <graham@grahamc.com>
+related-issues: 1 (initial process), 24 (implementation)
+---
+
+# Summary
+[summary]: #summary
+
+This RFC proposes an RFC Steering Committee who decide on a group of RFC
+shepherds for each RFC who guide the discussion to a general consensus and then
+propose a motion for a "Final Comment Period" (FCP) with a disposition for
+acception, rejection or postponing. (see Glossary for a short definition of
+these terms)
+
+
+# Motivation
+[motivation]: #motivation
+
+A lot of RFCs have stalled and already an [RFC has been submitted exactly on
+this topic](https://github.com/NixOS/rfcs/pull/18), which ironically has not
+been decided on either. This new RFC takes the above into account and tries to
+expand on that to flesh out the process further. During this effort a lot of
```

Comment:
Does it make any prescription for preexisting proposals (except closing #18 as superseded by the current one, I guess)?

Reply:
If this is accepted we will go through the past RFCs that have been decided on and follow the process as defined in the RFC. (i.e. #18 will most probably be either closed before we reach that point or else probably should be a rather easy one to reach consensus on closing)